config/config.go

```diff
@@ -158,7 +158,7 @@ func NewDefaultAgentConfig() *AgentConfig {
 			"rtprocess":   2 * time.Second,
 			"container":   10 * time.Second,
 			"rtcontainer": 2 * time.Second,
-			"connections": 3 * 60 * time.Minute,
+			"connections": 10 * time.Minute,
```

Comment:
is the overhead low enough that we can run this every 10-30 seconds?

Reply:
Ah good point - this was a typo. It should be `10 * time.Second`